pages.es/common/nc.md

```diff
@@ -1,32 +1,32 @@
 # nc
 
-> Netcat es una utilidad versátil para trabajar con datos TCP o UDP.
-> Más información: <https://manned.org/nc.1>.
+> Redirije datos de entrada o salida a un flujo de red a través de esta versátil herramienta.
+> Más información: <https://manned.org/man/nc.1>.
 
-- Escucha en un puerto determinado e imprime cualquier dato recibido:
+- Inicia un escuchador en un puerto TCP y enviale un archivo:
```

Comment:
- Inicia un escuchador en un puerto TCP para luego enviarle un archivo:

Reply:
Why? It's longer.

Reply:
* Inicia un escuchador en un puerto TCP y le envía un archivo: